content/en/database_monitoring/setup_mysql/gcsql.md

```diff
@@ -76,7 +76,7 @@ The Datadog Agent requires read-only access to the database in order to collect
 The following instructions grant the Agent permission to login from any host using `datadog@'%'`. You can restrict the `datadog` user to be allowed to login only from localhost by using `datadog@'localhost'`. See the [MySQL documentation][4] for more info.
 
 {{< tabs >}}
-{{% tab "MySQL ≥ 8.0" %}}
+{{% tab "MySQL ≥ 5.7" %}}
```

Comment:
Can you switch the order of the tabs in this section? Right now it shows the **MySQL ≥ 5.7** tab first, followed by **MySQL 5.6**. To ensure consistency with the other tabs in the article, I think it would be best if we listed **MySQL 5.6** first and then **MySQL ≥ 5.7**.